test/common.js

```diff
@@ -1,8 +1,8 @@
-//// Base file for unit test utilities
+// // Base file for unit test utilities
```

Comment:
Double comment makes the compiler ignore this twice, right? 😂

Reply:
:man_facepalming: 

Thanks eslint!

Couldn't change the rules so just added moar comments :slightly_smiling_face: